recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -123,3 +123,15 @@ def _cmake_target(self):
         if not self.options.shared:
             target += "-static"
         return target
+
+    @property
+    def _lib_target(self):
+        target = "paho-mqtt3"
+        target += "a" if self.options.asynchronous else "c"
+        if self.options.ssl:
+            target += "s"
+        if not self.options.shared:
+            # https://github.com/eclipse/paho.mqtt.c/blob/317fb008e1541838d1c29076d2bc5c3e4b6c4f53/src/CMakeLists.txt#L154
+            if tools.Version(self.version) < "1.3.1" or self.settings.os == "Windows":
```

Comment:
```suggestion
            if tools.Version(self.version) < "1.3.2" or self.settings.os == "Windows":
```

Reply:
Thank you, I put the link and did not follow it